pandas/core/base.py

```diff
@@ -393,7 +406,16 @@ def _agg(arg, func):
                 """
                 result = {}
                 for fname, agg_how in arg.items():
-                    result[fname] = func(fname, agg_how)
+                    # OWO CHANGES
```

Comment:
Perhaps encapsulate the changes in a new agg function to keep some consistency (agg_multi_d1? or something)